references/modified-prefixed-normalize.css

```diff
@@ -0,0 +1,502 @@
+/*! normalize.css v7.0.0 | MIT License | github.com/necolas/normalize.css */
+
+
+
+
+/* Document
+   ========================================================================== 
+   
+
+This is a modified version of normalize.css that is built-in to this library.  
+
+Modifications have been annotated, but only a minified version is kept in the code.
+
+   
+   
+*/
+
+
+
+/**
+ * ADDED From CSS Reset.
+ */
+
+.style-elements html, .style-elements body, .style-elements div, .style-elements span, .style-elements applet, .style-elements object, .style-elements iframe, .style-elements h1, .style-elements h2, .style-elements h3, .style-elements h4, .style-elements h5, .style-elements h6, .style-elements p, .style-elements blockquote, .style-elements pre, .style-elements a, .style-elements abbr, .style-elements acronym, .style-elements address, .style-elements big, .style-elements cite, .style-elements code, .style-elements del, .style-elements dfn, .style-elements em, .style-elements img, .style-elements ins, .style-elements kbd, .style-elements q, .style-elements s, .style-elements samp, .style-elements small, .style-elements strike, .style-elements strong, .style-elements sub, .style-elements sup, .style-elements tt, .style-elements var, .style-elements b, .style-elements u, .style-elements i, .style-elements center, .style-elements dl, .style-elements dt, .style-elements dd, .style-elements ol, .style-elements ul, .style-elements li, .style-elements fieldset, .style-elements form, .style-elements label, .style-elements legend, .style-elements table, .style-elements caption, .style-elements tbody, .style-elements tfoot, .style-elements thead, .style-elements tr, .style-elements th, .style-elements td, .style-elements article, .style-elements aside, .style-elements canvas, .style-elements details, .style-elements embed, .style-elements figure, .style-elements figcaption, .style-elements footer, .style-elements header, .style-elements hgroup, .style-elements menu, .style-elements nav, .style-elements output, .style-elements ruby, .style-elements section, .style-elements summary, .style-elements time, .style-elements mark, .style-elements audio, .style-elements video, .style-elements hr {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+}
+
+
+
+/**
+ * 1. Correct the line height in all browsers.
+ * 2. Prevent adjustments of font size after orientation changes in
+ *    IE on Windows Phone and in iOS.
+ */
+
+.style-elements html {
+  line-height: 1; /* 1 :NOTE: ADDED BY style-elements LIBRARY */
+  -ms-text-size-adjust: 100%; /* 2 */
+  -webkit-text-size-adjust: 100%; /* 2 */
+}
+
+/* Sections
+   ========================================================================== */
+
+/**
+ * Remove the margin in all browsers (opinionated).
+ */
+
+.style-elements body {
```

Comment:
I don't think this will do anything like this. You could reverse it though and add the class to the body tag. So `body.style-elements`, or just have a root `.style-elements` declaration to put the body and html root styles in. The assumption the would be that you need a `.style-elements` declaration on the body element for an SPA or the container object for a standalone component.